.github/workflows/containers.yml

```diff
@@ -39,6 +39,10 @@ jobs:
         timeout-minutes: 15
 
         services:
+            api-catalog-services-2:
```

Comment:
just to be sure, this instance will not register with the discovery service, right?

Reply:
It's not, it's disabled https://github.com/zowe/api-layer/pull/2751/files#diff-f32f969e996a1324801b0f00681c4741e170246e5902b36f06f605a46cf05240R30

Reply:
thanks